tests/VectorException/enc/sigill_handling.c

```diff
@@ -6,8 +6,22 @@
 #include <openenclave/internal/print.h>
 #include "../args.h"
 
-#include "../../../host/cpuid.h"
-#include "../../../host/linux/cpuid.c"
+// Wrapper over the CPUID instruction.
+void get_cpuid(
```

Comment:
This regresses in the direction where we had 3 separate implementations of CPUID in the codebase and IIRC the introduction of `oe_get_cpuid` was an attempt to prevent that. If we want to solve it this way, please resurrect #298 to track an alternate fix for that issue.

More generally, there needs to be a strategy for dealing with intrinsic dependencies so that OE can build cross-platform in a consistent way. A couple of observations:

- The host implementation is are currently being used in test enclave code. The use of inline assembly is not great, and will also have to be split again when compiling on Windows as PE enclaves come up. 
- The windows/Linux split is unavoidable even if we move to custom assembly based implementations due to different assemblers.
- Is there a value to having a subset of OE intrinsics in `common` that provide portable implementations?
    - `__builtin_ia32_rdrand32_step` will fail to build on Windows, for example, and there's an outstanding PR to consolidate those implementations as well (#972).

Reply:
@CodeMonkeyLeet  
Great points. As we discussed, I will create a new issue to keep track of the bigger picture re compiler intrinsics.
We have a few ideas that we can try out at the right time.
I will make a not in  #972 to use builtins instead of instrinsics since we are getting rid of system include paths in enclaves.